src/main/java/com/google/finapp/SpannerDaoInterface.java

```diff
@@ -0,0 +1,26 @@
+/*
+description goes here...
+ */
+
+package com.google.finapp;
+
+import com.google.cloud.ByteArray;
+import com.google.cloud.spanner.*;
+
+import java.math.BigDecimal;
+
+public interface SpannerDaoInterface {
+
+  void createCustomer(ByteArray customerId, String name, String address) throws SpannerException;
```

Comment:
Can we create a new exception type or are you planning to do that in a separate PR ?

Reply:
im going to do this in a separate PR